src/services/blockGraph.ts

```diff
@@ -46,6 +46,6 @@ export default class BlockGraphService {
   async getPreviousBlock (duration: any) {
     const currentTime = dayjs()
     const previousTime = currentTime.subtract(duration).unix()
-    return this.getBlockFromTimestamp(previousTime)
+    return await this.getBlockFromTimestamp(previousTime)
```

Comment:
you can delete the await here, as it's on the last line

Reply:
@leonprou Fixed